microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -135,36 +153,80 @@ To identify the culprit node in a detached tree that is causing the entire tree
 
 1. Select a different origin.
 
 The new origin is displayed in the **Detached Elements** tool.
 
    
 <!-- ====================================================================== -->
-## Additional considerations
+## About detached elements and memory leaks
+
+Detached elements aren't always an indication of a memory leak, and memory leaks aren't always caused by detached elements.  Memory leaks depend on the context of your application.
+
+A memory leak can occur in your application when an element is no longer attached to the Document Object Model (DOM) tree, but is still referenced by some JavaScript running on the webpage. These elements are called *detached elements*.  For the browser to garbage-collect (GC) the detached element, the element must not be referenced from the DOM tree or from JavaScript code.
+
+Memory issues affect webpage performance, including memory leaks, memory bloat, and frequent garbage collections.  Symptoms for your users include:
+
+*  The performance of a webpage gets progressively worse over time.
+*  The performance of a webpage is consistently bad.
+*  The performance of a webpage is delayed or appears to pause frequently.
+
+
+<!-- ====================================================================== -->
+## Run GC before getting detached elements, to show only elements that can't be GC'd
 
-When looking for memory leaks, remember that leaks can depend on the context of your application. For the demo application, you found detached elements that could not be garbage collected by the browser, and you identified the JavaScript that's retaining the detached elements. However, in the context of the demo application, it makes sense to retain the list of chat messages, so that if a user switches back to **Room 1**, the message log is preserved.
+To only show the elements that are detached from the DOM tree and that can't be garbage-collected, always click **Collect Garbage** first, and then click **Get Detached Elements**.
+
+Even though some elements might appear as detached at one point in time, you won't know whether they are actually still referenced by JavaScript code in the webpage until you run GC.  When switching between chat rooms in the demo app, the web page removes the elements that are used to display messages from the DOM.  But switching to a different instance of the `Room` class doesn't delete these elements entirely and references to these elements still exist, so, these elements stay in memory.
+
+
+<!-- ====================================================================== -->
+## Re-attaching elements
+
+To use the demo webpage, you find detached elements that can't be garbage-collected by the browser, and you identify the JavaScript that's retaining the detached elements.  However, in the context of the demo webpage, it makes sense to retain the list of chat messages, so that if a user switches back to **Room 1**, the message log is preserved.
```

Comment:
Can we use the same content that's in dom-leaks-memory-tool-detached-elements.md here? I find it clearer.